cmd/containerd-shim-runhcs-v1/exec_hcs.go

```diff
@@ -300,7 +300,11 @@ func (he *hcsExec) Kill(ctx context.Context, signal uint32) error {
 		}
 		return nil
 	case shimExecStateExited:
-		return errors.Wrapf(errdefs.ErrNotFound, "exec: '%s' in task: '%s' not found", he.id, he.tid)
+		log.G(ctx).WithFields(logrus.Fields{
+			"eid": he.id,
+			"tid": he.tid,
+		}).Debug("shimExecStateExited")
+		return nil
```

Comment:
Doesn't this change violate [this invariant](https://github.com/microsoft/hcsshim/blob/f50f9750ce18de430c47bff1f16fa855daaf124e/cmd/containerd-shim-runhcs-v1/exec.go#L54)?
I don't know if we rely on that property anywhere though